dataeng/resources/snowflake-replica-import.sh

```diff
@@ -2,18 +2,34 @@
 
 env
 
-${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
- ImportMysqlDatabaseFromS3ToSnowflakeSchemaTask --local-scheduler \
- --date $(date +%Y-%m-%d -d "$RUN_DATE") \
- --credentials $SNOWFLAKE_CREDENTIALS \
- --warehouse $WAREHOUSE \
- --role $ROLE \
- --sf-database $SNOWFLAKE_DATABASE \
- --schema $SCHEMA \
- --scratch-schema $SCRATCH_SCHEMA \
- --run-id $BUILD_ID \
- --database $DATABASE \
- --overwrite \
- ${INCLUDE} \
- ${EXCLUDE} \
- ${EXTRA_ARGS}
+# Only invoke this script on Mondays, or when forced to run via the FORCE
+# parameter, otherwise skip it.  Since this job is triggered by an upstream for
+# which we can't change the cron schedule (due to vertica not yet being
+# deprecated), we must skip it manually in this shell script.
+
+DAY_OF_WEEK=$(date +%u)
+MONDAY=1
+if [[ ${DAY_OF_WEEK} -eq ${MONDAY} -o ${FORCE} = "true" ]]; then
```

Comment:
Non-blocker: Can we use conditional build step instead of putting this in the script ?
https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.step.MultiJobStepContext.conditionalSteps

Reply:
I briefly evaluated that option, but the only condition function they provided was `time()` which was not sufficient:

```
// Only runs the build steps during a certain period of the day.
time(int earliestHours, int earliestMinutes, int latestHours, int latestMinutes, boolean useBuildTime) 
```